docker.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+command=$1
+IMAGE=badi-events-backend
+
+help() {
+	messages=(
+		"docker.sh <command>\n"
+		"\t- help / --help: Displays this message\n"
+		"\t- build: builds the Docker image\n"
+		"\t- container: shows the current container name\n"
+		"\t- enter: go to the bash terminal in the container\n"
+		"\t- start: starts the docker container in the background\n"
+		"\t- stop: stops the docker container\n"
+		"\n"
+	)
+	printf "${messages[*]}"
+}
+
+error() {
+	local error_message=$1
+	printf "Error: $error_message\n\n"
+	help
+	exit 1
+}
+
+build() {
+	docker build -t $IMAGE .
+}
+
+enter() {
+	docker exec -it $(get_container_name) /bin/bash
+}
+
+get_container_name() {
+	echo $(docker container ls | grep $IMAGE | awk '{print $(NF)}')
+}
+
+start() {
+	docker run -d -v `pwd`:/app -p 8080:8080 $IMAGE
```

Comment:
So I know that I said that that would be my last comment and that I would then merge in this pull request, but I tested out this shell script and I had a couple of suggestions.  So I would use the start command to run the Docker image in a Docker container and then would use the stop command to stop the Docker container.  If I ran the `docker ps` command then I would not see any running containers, but if I ran the `docker ps -a` command then I would see all of the Docker containers that I had stopped.  One thing I would always add to my `docker run` commands is the `--rm` option so that the Docker container would get automatically removed once it got stopped.  Not sure if you meant to be able to start and run multiple Docker containers, but I am not sure if it would work anyways as anymore than one Docker container would conflict on the 8080 port that they are getting assigned to.  Also if you only really want one Docker container running at a time then you could also just set the `--name` option with the `docker run` command and just manually assign a name to the Docker container that you can set as a variable in this script as you did the name for the Docker image and use that everywhere where the Docker container is referenced.

Reply:
That's super useful info. I was just using it because I was being lazy and didn't feel like typing out the full commands. But perhaps its better to just remove it for now and if we need something like it, we can build it more to be more robust then.